TA-puppet-alert-actions/bin/inputs/orchestrator_actions.py

```diff
@@ -0,0 +1,135 @@
+# $SPUNK_HOME/etc/apps/TA-puppet-alert-actions/bin/inputs/orchestrator_actions.py
+#
+import sys
+import os
+#
+#
+# If Splunk was installed in a custom location, change SPLUNK_APPS to the path where apps are installed.
+APP_NAME = 'TA-puppet-alert-actions'
+if os.name == 'nt':
+  SPLUNK_APPS = 'C:\Program Files\Splunk\etc\/apps\/'
+else:
+  SPLUNK_APPS = '/opt/splunk/etc/apps'
+PIE_LIB = os.path.join(SPLUNK_APPS, APP_NAME, 'bin', 'ta_puppet_alert_actions')
+AOB_LIB = os.path.join(SPLUNK_APPS, APP_NAME, 'bin', 'ta_puppet_alert_actions', 'aob_py3')
+#
+#
+# Here we modify the system path for Python to find the required helper libs.
+try:
+  PATHS = [PIE_LIB,AOB_LIB]
+  for path in PATHS:
+    sys.path.append(path)
+  import pie
+  from splunk_aoblib.setup_util import Setup_Util
+except Exception as e:
+  sys.stderr.write("TA-puppet-alert-actions: Failed to import required libs - {}".format(e))
+#
+#
+# Define Splunk URI and session key.
+uri = "https://localhost:8089"
+session_key = sys.stdin.readline().strip()
```

Comment:
This is the splunky way for scripted inputs to grab a Splunk session token to auth to the endpoints required to get the configured user credentials. Per the `inputs.conf` file, when Splunk runs this script it will generate a token for the `splunk-system-user` and pass it in.